### PR TITLE
Basic PR after pipeline re-creation

### PR DESCRIPTION
**Pull request by WackerO**
This is the basic PR for the general pipeline code after I created it with nf-core create; no actual code changes

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/differentialabundance/differentialabundance/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by pinin4fjords** [comment]
Could we merge https://github.com/nf-core/differentialabundance/pull/1 first (I don't have permissions)? I feel like it's fair that conflicts get resolved in the later PRs :-)

**Comment by WackerO** [comment]
Yes sure! I think I cannot give you permissions, @ggabernet probably has to do that

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
-# differentialabundance/differentialabundance: Contributing Guidelines
+# nf-core/differentialabundance: Contributing Guidelines
```
Comment (ggabernet) [inline .github/CONTRIBUTING.md:1]: It's a bit weird that here everywhere was `differentialabundance`, also in the template branch so either everything would need to be created again from scratch with the `nf-core create` command, or at least just the template branch.

**Review by ggabernet — APPROVED** [review]
LGTM, the template branch will need to be updated as well

**Comment by WackerO** [comment]
@nf-core-bot fix linting

**Pull request merged**